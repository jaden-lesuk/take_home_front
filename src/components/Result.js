import React from 'react'

export default function Result(props) {
    return (
        <div style={styles.resultStyle}>
            {
                props.weather === "err"?
                <h2>Please make sure the city name is correct</h2>
                : <h2>Hmmm, the condition seems to be:  {props.weather}</h2>
            }
        </div>
    )
}

const styles = {
    resultStyle: {
        textAlign: 'center',
        color: 'white',
    }
}