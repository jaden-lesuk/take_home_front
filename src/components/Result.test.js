import React from 'react';
import Result from './Result';
import { create } from 'react-test-renderer'

describe('Result Component Snapshot Test',()=>{
    test('testing result', () => {
    let tree = create(<Result />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})
