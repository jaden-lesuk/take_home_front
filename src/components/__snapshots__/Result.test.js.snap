// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result Component Snapshot Test testing result 1`] = `
<div
  style={
    Object {
      "color": "white",
      "textAlign": "center",
    }
  }
>
  <h2>
    Hmmm, the condition seems to be:  
  </h2>
</div>
`;
